DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -243,16 +239,16 @@ FROM Centres
             return ipPrefixes ?? new string[0];
         }
 
-        public (bool autoRegistered, string? autoRegisterManagerEmail) GetCentreAutoRegisterValues(int centreId)
-        {
-            return connection.QueryFirstOrDefault<(bool, string?)>(
-                @"SELECT AutoRegistered, AutoRegisterManagerEmail
-                        FROM Centres
-                        WHERE CentreID = @centreId",
-                new { centreId }
-            );
-        }
-
+        public (bool autoRegistered, string? autoRegisterManagerEmail) GetCentreAutoRegisterValues(int centreId)

```

Comment:
Is this line endings? If so, can you please highlight this to Ibrahim so he knows he's causing it? 🙏